.travis.yml

```diff
@@ -44,4 +45,7 @@ matrix:
         #       but since those are not supported by TravisCI, we need to install targets manually.
         - unzip "/c/Program Files (x86)/Windows Kits/10/Vsix/WDK.vsix" -d wdk
         - cp -R wdk/\$VCTargets/* "$MSVC/Common7/IDE/VC/VCTargets"
-        - MSBuild.exe haxm-windows.vcxproj //p:Configuration="Debug" //p:Platform="x64"
+        - export PROPS="//p:DefaultWindowsSDKVersion=10.0.17763.0 //p:SpectreMitigation=false"
```

Comment:
This Windows SDK version number is slightly different from the one at line 35 above (`.0` vs. `.1`). I just checked my local installation, and I saw the same discrepancy: Apps & Features reports that I've installed Windows SDK 10.0.17763.1, but under `C:\Program Files (x86)\Windows Kits\10\bin\` there's no folder named `10.0.17763.1`, only `10.0.17763.0`. Maybe that's why `.1` wouldn't work here?

Reply:
It seems Microsoft versioning assigns the last integer to represent minor fixes/patches with no functionality change. Consequently, the Windows SDK version specified in the project always ends with `.0`. In fact, when I try to change this setting from Visual Studio, it offers me the options (based on local Windows SDK installations) `8.1` and `10.0.17134.0`, despite the fact that I have `10.0.17134.12` installed.